appshell/mac/English.lproj/MainMenu.xib

```diff
@@ -76,6 +76,12 @@
                                     <action selector="performZoom:" target="-1" id="240"/>
                                 </connections>
                             </menuItem>
+                            <menuItem title="Enter Full Screen" keyEquivalent="f" id="241">
```

Comment:
Probably a very minor one - the documentation states to add to the View menu - 

https://developer.apple.com/library/mac/documentation/Cocoa/Reference/ApplicationKit/Classes/NSWindow_Class/#//apple_ref/occ/instm/NSWindow/

If an application supports fullscreen, it should add a menu item to the View menu with toggleFullScreen: as the action, and nil as the target.